### ISSUE TITLE
No linebreaks in subtitle?

### ISSUE THREAD

**Issue by GrosserStuhl** [elai950/AlertToast#37]
**Describe the bug**
I assume it is a bug and not intention. I cannot present a message that is longer than X characters.
I am using a `Banner` type but it seems to behave similar with `HUD` too.

My assumption was that using a subtitle, it would show multiple lines and the banner would grow bigger in vertical size.

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/6041146/151051722-4c7968b8-4d93-4c75-8fa1-701ff56bce32.png)

**Smartphone (please complete the following information):**
 - Device: Simulator - iPhone 11
 - OS: iOS 15
 - Version master branch

**Comment by aledebla03** [comment]
Same here, also in title

**Comment by AnmolMalpani** [comment]
Title going out of the screen
<img width="392" alt="Screenshot 2022-03-10 at 5 22 59 PM" src="https://user-images.githubusercontent.com/39443206/157656626-3df8bd96-0f21-44a2-aca4-5634d320849f.png">
.

**Comment by RudraSen2** [comment]
Try Adding `<br>` to the text if you're using SwiftUI 3.

**Comment by elai950** [comment]
Hey @GrosserStuhl ,
I tried to reproduce the issue on my side with no luck.

As you can see in the picture this very long string does linebreak where it needs to.
I'll close this issue for that reason but please keep me updated if you find the cause of this problem. I'm curious.

```swift
.toast(isPresenting: $showBannerRegular, duration: 4){
                    AlertToast(displayMode: .banner(.slide), type: .regular, title: "Welcome! 👋", subTitle: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.", style: .style(backgroundColor: .blue))
                }
```
<img width="332" alt="צילום מסך 2023-01-31 ב-10 21 43" src="https://user-images.githubusercontent.com/37900883/215706055-560dfc00-7c3f-41d8-b3ca-d62170c10af8.png">

**Comment by harleylovitos** [comment]
I'm having this issue here. It was working last 2nd or 1st week of January but now it showing like this.
![IMG_1886](https://user-images.githubusercontent.com/2785944/217878258-e0641b94-a973-4630-882f-2394a1915c6c.jpg)

**Comment by elai950** [comment]
> I'm having this issue here. It was working last 2nd or 1st week of January but now it showing like this.
> 
> ![IMG_1886](https://user-images.githubusercontent.com/2785944/217878258-e0641b94-a973-4630-882f-2394a1915c6c.jpg)
> 
> 

From previous tests, it might be because you didn't specified with to the view that the toast modifier is on him.
Try give him a frame width of even infinity in a vstack and see if it solved.

Usually this problem occurring when there is so width to the view.
Generally speaking in SwiftUI I mean.

**Comment by harleylovitos** [comment]
I added this code and same problem.
`ZStack
{VStack{
 some code....
}.padding(20)
                .frame(maxWidth: .infinity)
                .toast(isPresenting: self.$showToast,duration: 5, tapToDismiss: true, alert: {
                    AlertToast(displayMode: .banner(.slide),type: self.toastType, title: self.toastMessage)
                }, onTap: {
                   some codes...
                }, completion: {
                   some codes...
 })
 }`
                 
I also tried `.frame(width: .infinity)`

![IMG_1889](https://user-images.githubusercontent.com/2785944/217886097-3b06bb3c-acbd-4758-8198-802e401a88c0.jpg)

**Comment by elai950** [comment]
This bug was fixed in 1.3.9.
Please check.

**Comment by harleylovitos** [comment]
Thank you :)
